server/WebAPI/Services/EmailSender.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using MailKit.Net.Smtp;
+using MimeKit;
+using WebAPI.DTO;
+using WebAPI.Interfaces;
+
+namespace WebAPI.Services
+{
+  public class EmailSender : IEmailSender
```

Comment:
is this just duplicate of 
server/Service/Services/EmailSender.cs 
if so, why do we need it?

Reply:
Yes, they are duplicates. came by accident and have now been removed